internal/server/get_payments_test.go

```diff
@@ -217,7 +235,9 @@ func TestGetPaymentsWhenTemplateErrors(t *testing.T) {
 		Return(caseItem, nil)
 	client.
 		On("RefDataByCategory", mock.Anything, sirius.PaymentSourceCategory).
-		Return(paymentSources, nil)
+		Return(paymentSources, nil).
+		On("RefDataByCategory", mock.Anything, sirius.PaymentReferenceType).
+		Return(referenceTypes, nil)
```

Comment:
Just missing a test now where the payment reference types api call returns an error